modules/programs/vim.nix

```diff
@@ -166,6 +193,7 @@ in {
     programs.vim = {
       package = vim;
       plugins = defaultPlugins;
+      optionalPlugins = defaultOptionalPlugins;
```

Comment:
Not sure if we need to set these options here considering that they have a default value already.